sdk/contentsafety/azure-ai-contentsafety/samples/sample_manage_blocklist.py

```diff
@@ -39,7 +37,30 @@ def create_or_update_text_blocklist():
         print(e)
         raise
 
+    # [END create_or_update_text_blocklist]
+
 def add_block_items():
+    # [START add_block_items]
+
+    import os
+    from azure.ai.contentsafety import ContentSafetyClient
+    from azure.core.credentials import AzureKeyCredential
+    from azure.ai.contentsafety.models import (
+        TextBlockItemInfo,
+        AddBlockItemsOptions
+    )
+    from azure.core.exceptions import HttpResponseError
+
+    key = os.environ["CONTENT_SAFETY_KEY"]
+    endpoint = os.environ["CONTENT_SAFETY_ENDPOINT"]
+
+    # Create an Content Safety client
+    client = ContentSafetyClient(endpoint, AzureKeyCredential(key))
```

Comment:
Just a suggestion, you dont need to include this in each of your code snippets since you already showcase how to create a client in the readme.

Reply:
I'd like to keep it now to provide a complete runnable code for customer to copy directly.